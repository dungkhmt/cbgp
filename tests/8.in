50 50
50 80
14 3
49 42
36 15
19 11
27 6
47 17
47 2
21 5
28 7
34 29
32 26
37 4
20 5
29 14
4 5
32 13
43 0
6 22
49 36
18 20
43 10
32 10
6 2
24 0
41 6
28 1
36 41
48 24
32 25
45 4
30 9
20 23
38 6
24 7
35 45
49 39
9 4
9 12
15 0
25 9
5 8
26 11
3 0
40 15
34 16
46 39
44 26
32 33
31 9
47 7
37 45
27 28
32 11
29 3
24 2
2 3
35 37
34 14
32 9
27 1
22 14
3 16
13 10
42 18
18 5
17 7
42 5
11 16
42 15
12 0
43 19
30 23
27 36
36 1
26 4
32 31
11 3
43 33
18 23
6 3
1
30 5 71 2 60 3 57 6 29 21 4 4 63 11 66 2 22 22 38 13 36 8 48 7 36 11 37 9 52 11 26 1 49 8 64 3 18 5 37 7 15 6 0 0 57 16 16 7 33 2 39 11 31 23 70 4 68 2 53 9 29 20 39 14 45 8 37 10 50 9 25 22 76 0 27 22 42 32 4 0 28 2 64 12 22 1 34 7 40 32 25 25 2 0 63 5 30 2 21 0