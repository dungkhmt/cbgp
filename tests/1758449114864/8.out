(13, 2), (48, 41), (35, 14), (18, 10), (26, 5), (46, 16), (46, 1), (20, 4), (27, 6), (33, 28), (31, 25), (36, 3), (19, 4), (28, 13), (3, 4), (31, 12), (42, -1), (5, 21), (48, 35), (17, 19), (42, 9), (31, 9), (5, 1), (23, -1), (40, 5), (27, 0), (35, 40), (47, 23), (31, 24), (44, 3), (29, 8), (19, 22), (37, 5), (23, 6), (34, 44), (48, 38), (8, 3), (8, 11), (14, -1), (24, 8), (4, 7), (25, 10), (2, -1), (39, 14), (33, 15), (45, 38), (43, 25), (31, 32), (30, 8), (46, 6), (36, 44), (26, 27), (31, 10), (28, 2), (23, 1), (1, 2), (34, 36), (33, 13), (31, 8), (26, 0), (21, 13), (2, 15), (12, 9), (41, 17), (17, 4), (16, 6), (41, 4), (10, 15), (41, 14), (11, -1), (42, 18), (29, 22), (26, 35), (35, 0), (25, 3), (31, 30), (10, 2), (42, 32), (17, 22), (5, 2), 
numberIntersection=18 
minAngle=0.03634762101902034 
sumAngle=53.59233097806343 
minEdgeLength=1.0 
sumEdgeLength=3979.8835894434324 
minDistance=0.02933257721442166 
sumDistance=254242.75791120317 

-18.0000000000 0.0363476210 53.5923309781 1.0000000000 3979.8835894434 0.0293325772 254242.7579112032
0: (45, 91),
1: (27, 2),
2: (55, 2),
3: (129, 30),
4: (89, 200),
5: (0, 0),
6: (24, 35),
7: (0, 89),
8: (21, 115),
9: (25, 50),
10: (50, 19),
11: (0, 2),
12: (26, 53),
13: (110, 0),
14: (87, 63),
15: (57, 13),
16: (6, 1),
17: (71, 93),
18: (23, 3),
19: (62, 97),
20: (0, 108),
21: (1, 0),
22: (63, 85),
23: (35, 17),
24: (50, 74),
25: (61, 25),
26: (43, 89),
27: (80, 60),
28: (67, 14),
29: (28, 110),
30: (51, 76),
31: (70, 56),
32: (22, 3),
33: (67, 16),
34: (170, 44),
35: (22, 114),
36: (199, 0),
37: (22, 113),
38: (116, 14),
39: (115, 29),
40: (0, 1),
41: (105, 82),
42: (21, 2),
43: (63, 15),
44: (199, 177),
45: (76, 26),
46: (25, 2),
47: (46, 17),
48: (136, 8),
49: (200, 200),
45 91 27 2 55 2 129 30 89 200 0 0 24 35 0 89 21 115 25 50 50 19 0 2 26 53 110 0 87 63 57 13 6 1 71 93 23 3 62 97 0 108 1 0 63 85 35 17 50 74 61 25 43 89 80 60 67 14 28 110 51 76 70 56 22 3 67 16 170 44 22 114 199 0 22 113 116 14 115 29 0 1 105 82 21 2 63 15 199 177 76 26 25 2 46 17 136 8 200 200 
0: (45, 91), 1: (27, 2), 2: (55, 2), 3: (129, 30), 4: (89, 200), 5: (0, 0), 6: (24, 35), 7: (0, 89), 8: (21, 115), 9: (25, 50), 10: (50, 19), 11: (0, 2), 12: (26, 53), 13: (110, 0), 14: (87, 63), 15: (57, 13), 16: (6, 1), 17: (71, 93), 18: (23, 3), 19: (62, 97), 20: (0, 108), 21: (1, 0), 22: (63, 85), 23: (35, 17), 24: (50, 74), 25: (61, 25), 26: (43, 89), 27: (80, 60), 28: (67, 14), 29: (28, 110), 30: (51, 76), 31: (70, 56), 32: (22, 3), 33: (67, 16), 34: (170, 44), 35: (22, 114), 36: (199, 0), 37: (22, 113), 38: (116, 14), 39: (115, 29), 40: (0, 1), 41: (105, 82), 42: (21, 2), 43: (63, 15), 44: (199, 177), 45: (76, 26), 46: (25, 2), 47: (46, 17), 48: (136, 8), 49: (200, 200), 
numberIntersection=18 minAngle=0.03634762101902034 sumAngle=53.592330978012896 minEdgeLength=1.0 sumEdgeLength=3979.883589443434 minDistance=0.02933257721442166 sumDistance=254242.75791120142 
Time taken: 29.542 s
